EKF/ekf_helper.cpp

```diff
@@ -1623,17 +1622,17 @@ void Ekf::calcExtVisRotMat()
 
 	// convert filtered vector to a quaternion and then to a rotation matrix
 	q_error.from_axis_angle(_ev_rot_vec_filt);
-	_ev_rot_mat = quat_to_invrotmat(q_error);
+	q_error.normalize();
+	_ev_rot_mat = Dcmf(q_error); // rotation from EV reference to EKF reference
```

Comment:
There is already a PR dealing with how the quat_to_invrotmat function is used - see https://github.com/PX4/ecl/pull/635